src/spirc.rs

```diff
@@ -77,7 +78,7 @@ impl<D: SpircDelegate> SpircManager<D> {
 
             repeat: false,
             shuffle: false,
-            volume: 0x8000,
+            volume:	32767,
```

Comment:
There are now two variables tracking the volume, and they may have different values (at startup, until a Volume command is received).
Could you remove the one in SpircManager and only rely on the Player one